sdk/identity/azure-identity/tests/test_auth_file_credential.py

```diff
@@ -0,0 +1,26 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+import os
+
+from azure.identity import AuthFileCredential
+from azure.core.exceptions import ClientAuthenticationError
+
+def test_auth_file_credential_parse():
```

Comment:
It would be better to test the credential's public API. That is to say, construct the credential with the mock file, then call its `get_token` method and validate the credential's HTTP requests ([for example](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/identity/azure-identity/tests/test_client_secret_credential.py#L46)).